src/include/misc.h

```diff
@@ -6,6 +6,26 @@
  * See the file LICENSE for redistribution information.
  */
 
+#ifndef __WT_MISC_H
+#define __WT_MISC_H
+
+/*
+ * When compiling for code coverage measurement it is necessary to ensure that inline functions in
+ * header files that are #included in multiple source files are not inlined.
+ *
+ * Otherwise, it is possible that there will be multiple copies of the function across the linked
+ * executable with the result that the code coverage counts for branch coverage (both in terms of
+ * branches in the code and the number of branches executed) will be too high and incorrect.
+ *
+ * In non-code coverage builds, preventing inlining would impact performance and so this must only
+ * take place when performing code coverage.
+ */
+#ifdef CODE_COVERAGE_MEASUREMENT
```

Comment:
Consider adding a preprocessor conditional guard to ensure this is not enabled for a windows build. I know we don't perform code coverage on windows, it would be to guard against an inadvertent configuration.

Reply:
That's a good suggestion and easy to add.

Reply:
Done!